scripts/python-wheels-windows.bat

```diff
@@ -40,7 +37,7 @@ pushd %ARROW_SRC%
 @rem fix up symlinks
 git config core.symlinks true
 git reset --hard || exit /B
-git checkout %pyarrow_commit% || exit /B
+git checkout "apache-arrow-%pyarrow_version%" || exit /B
```

Comment:
Don't think you want this because we may use this repo to build untagged commits

Reply:
Fine by me, we can just use the ref syntax, which includes tags and commits